website/src/utils/isPost.ts

```diff
@@ -0,0 +1,31 @@
+import { isStringArray } from './isStringArray';
```

Comment:
I think libraries like Zod can help better structure this - rolling your own, it's easy to get out of sync with the model if you change, add, or remove a field.

Reply:
Thanks, I'll check out Zod! 

What do you mean by a field though? Do you mean as in the field of our interface, Post?